NOTICE

```diff
@@ -3,11 +3,11 @@ Copyright 2009-2020 The VOTCA Development Team
 
 This product includes software developed at
 The VOTCA Development Team (http://www.votca.org)
-and at Juelich Supercomputing Centre.
+and at Forschungszentrum Juelich GmbH, Germany.
 
 This software contains code, which was
 derived from VOTCA KMC, MOO and CTP project.
 
 The implementation of the Ewald method is contributed
-by Rene Halver, Godehard Sutmann from Juelich Supercomputing
-Centre.
+by Rene Halver, Godehard Sutmann from Forschungszentrum 
+Juelich GmbH, Germany.
```

Comment:
@rhalver How about this mention in the NOTICE file?

Reply:
I think that should be fine, I will change the copyrights in the source files and will update them.

Reply:
It seems I am not allowed to push the changes to the branch? Errors out with 403.

Reply:
I added you to the xtp group. How about now?

Reply:
Just pushed the changes, thanks.